src/components/RoomSelector.vue

```diff
@@ -194,6 +182,9 @@ export default {
 			this.$root.$emit('close')
 			this.$emit('close')
 		},
+		onSelect(item) {
+			this.selectedRoom = item.token
+		},
 		select() {
 			this.$root.$emit('select', this.selectedRoom)
```

Comment:
Please check why it is needed to be emitted from the root or if EventBus can be an alternative here

Reply:
That was there historically: https://github.com/nextcloud/spreed/blob/0c9599e598ca34e6b0ba27dae3cb319a668be7fd/src/deck.js#L103
If you ask me, I'd prefer this way, cause EventBus spins an extra Vue instance and add some weight to the bundle, which we want to avoid for plugins, if possible

Reply:
Maybe it is not needed in the first place because select here is meant to pass the info to the parent component only ( and from their the select button).